README.md

```diff
@@ -13,3 +13,19 @@ This is a simple cross-platform OpenGL engine.
 ### Build and Run
 1. Run `cmake --build .` to build
 2. Run `./main` to run
+
```

Comment:
no need to update readme.md. Logger is a simple feature, we don't need to mention it